tests/foreman/ui/test_setting.py

```diff
@@ -556,6 +557,50 @@ def test_negative_update_foreman_url_param(self):
                     self.assertNotEqual(param_value, self.saved_element)
 
     @tier1
+    def test_positive_update_login_page_footer_text(self):
+        """Updates parameter "Login_page_footer_text" under General tab
+
+        :id: 56a983c4-925f-4cbe-8fdb-ce344219d739
+
+        :expectedresults: Parameter is updated successfully
+
+        :CaseImportance: Critical
+        """
+        self.tab_locator = tab_locators['settings.tab_general']
+        self.param_name = 'login_text'
+        with Session(self.browser) as session:
+            self.original_value = self.settings.get_saved_value(
+                self.tab_locator, self.param_name)
+            for param_value in valid_data_list():
+                with self.subTest(param_value):
+                    edit_param(
+                        session,
+                        tab_locator=self.tab_locator,
+                        param_name=self.param_name,
+                        param_value=param_value,
+                    )
+                    self.saved_element = self.settings.get_saved_value(
+                        self.tab_locator, self.param_name)
+                    self.assertEqual(param_value, self.saved_element)
+
+    @stubbed
+    @tier1
+    def test_negative_update_login_page_footer_text(self):
+        """ Updates parameter "Login_page_footer_text" under General tab
```

Comment:
@abalakh : Hey, there is discussion going on to tackle the -ve scenario. Ideally when user update the footer w/ long string then whole UI gets distorted. This can be automated easily but we are not sure if its going to be fixed w/ string length or long string going to be stripped off.